islandora_book.module

```diff
@@ -361,3 +361,33 @@ function islandora_book_islandora_solution_pack_child_relationships($cmodels) {
     ];
   }
 }
+
+/**
+ * Implements hook_help().
+ */
+function islandora_book_help($route) {
+  switch ($route) {
+    // Main help page for the module.
+    case 'help.page.islandora_book':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Islandora Book Solution Pack allows users to add and manage Book objects and their Page objects. There are various viewers that can be used for Books and Pages, and the desired viewer can be chosen at www.yoursite.com/admin/config/islandora/solution_pack_config/book. See the <a href=":wiki">Islandora Book Solution Pack wiki*</a> for more information.', [
```

Comment:
Can we make `www.yoursite.com/admin/config/islandora/solution_pack_config/book` a link?